packages/amplify-provider-awscloudformation/src/pre-push-cfn-processor/cfn-pre-processor.ts

```diff
@@ -24,3 +39,176 @@ export async function preProcessCFNTemplate(filePath: string): Promise<string> {
   await writeCFNTemplate(cfnTemplate, newPath, { templateFormat });
   return newPath;
 }
+
+//get data from custom polcies file and write custom policies to CFN template
+export async function writeCustomPoliciesToCFNTemplate(
+  resourceName: string,
+  service: string,
+  resourceDir: string,
+  cfnFile: string,
+  category: string,
+  filePath: string
+) {
+  if (!(service === 'Lambda' || service === 'ElasticContainer')) {
```

Comment:
recommend changing to:
```suggestion
  if (!customPolicySupportedServices.includes(service)) {
```
then define
```
const customPolicySupportedServices = ['Lambda', 'ElasticContainer']
```
at the top of the file. This should make it easier to extend this feature to other services in the future